modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/json/content/ContentPublishInfoJson.java

```diff
@@ -36,4 +37,26 @@ public Instant getFirst()
     {
         return first;
     }
+
+    @Override
```

Comment:
It is often unnecessary  and even error-prone to write equals hashcode for DTOs.